Automattic/woocommerce-payments
# Fix - Jurassic Tube Setup Script Directory Creation

#### Changes proposed in this Pull Request

While setting up Jurassic Tube tunnel, the setup script tries to create the directories `bin/jt` under `docker`. However, the directory creation fails terminating the script since it involves creating a parent and a child directory.

This PR fix the problem by instructing `mkdir` to create both the parent & child directories.

![Screenshot](https://d.pr/i/759oym+)

#### Testing instructions

* Checkout `develop`, install dependencies & run the script `npm run setup:jurassic-tube`. (Make sure you haven't set this up earlier. Ideally run in a fresh copy of the repo).
* Notice that the script errors while creating the required directories.
* Checkout the branch `fix/jurassic-tube` & run the script `npm run setup:jurassic-tube` again.
* Notice that the script executes as expected and the tunnel is set up.

-------------------
